src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -57,6 +57,24 @@ public static UserStore getTestInstance(PersistentStorageAgent persistentStorage
 
   /** The in-memory list of Users. */
   private List<User> users;
+  
+  /** Access the current amount of users **/
+  public int getNumUsers() {
+	  return users.size();
+  }
+  
+  public String getNewest() {
```

Comment:
Should this return the name of the newest user or maybe the `User` object itself?

Reply:
Returning the User object would be useful to access more attributes about the Newest User but right now i think it's fine to return only the name

Reply:
Yep it should, that would be better for testing too